src/pages/Halving/styled.tsx

```diff
@@ -0,0 +1,161 @@
+import styled from 'styled-components'
```

Comment:
I would suggest not using styled-components because it's hard to maintain. But it's just a suggestion.